.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Why this file? Should be ignored, right?

Reply:
Thanks!